### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by saifudinbadani**
Navigation Component added.

Screen Shot 2022-03-18 at 6 03 57 pm

### CODE REVIEW

File: src/css/cart.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import "../common.css";
```
Comment (ivinayakg) [inline src/css/cart.css:1]: just  a suggestion, dont add files which are unrelated to the PR.

File: src/components/Homepage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import '../css/home-page.css';
+import { HomePageCategories, HomePageCards, HomePageHero } from '../components/Homepage/';
+
+
+const HomePage = () => {
+
```
Comment (ivinayakg) [inline src/components/Homepage.js]: make a folder directly under src and name it pages, in that folder make your pages , like this file.

Comment (ivinayakg) [reply src/components/Homepage.js]: will result in a better file structuring

File: src/components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import '../css/nav.css';
+
+const Navigation = () => {
+    return  <nav class="nav-container display-flex primary-clr p-rl-6">
+        <a href="./index.html" class="nav-link font-size-2pt6">
```
Comment (ivinayakg) [inline src/components/Navigation.js]: dont use anchor tag(<a>) as it will result in a non-SPA application which is not a desired outcome we want when using react and react-router-dom use Link component from react-router-dom instead

**Review by ivinayakg — COMMENTED** [review]
focus a bit more on the file structuring as this structuring which you have right now may hinder when you scale the app. apart from it the navbar is clean and decent need one change though but is good 👍

File: src/components/Navigation.js (hunk 3 above)
Comment (saifudinbadani) [reply src/components/Navigation.js]: Yes, i will do that after adding all of the components at one go.

File: src/components/Homepage.js (hunk 2 above)
Comment (saifudinbadani) [reply src/components/Homepage.js]: Yes, will do that, easier to access and read.

File: src/css/cart.css (hunk 1 above)
Comment (saifudinbadani) [reply src/css/cart.css:1]: this files was needed to render navigation.

File: src/css/cart.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+.cart-container{
+    margin-top: 8rem;
+}
+
+.card-items-qty-input{
+    width: 2rem;
```
Comment (samyakshah3008) [inline src/css/cart.css:8]: can give max or min-width for responsiveness, can add later on:)

File: src/css/login-signup.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+.form-control > label {
+    color: var(--text-on-white);
+}
+
+.container {
+    margin-top: 10%;
+    margin-left: 35%;
+    background-color: #fff;
+    border-radius: 5px;
+    box-shadow: 0 20px 10px rgba(0, 0, 0, 0.3);
```
Comment (samyakshah3008) [inline src/css/login-signup.css:24]: you can use css variables for color here as well

File: src/pages/Homepage.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+const HomePage = () => {
+
+    return <div className='home-page-body bg-clr'>
+        <Navigation />
+        <HomePageCategories />
+        <HomePageHero />
+        <HomePageCards />
+        </div>
+    
+
```
Comment (samyakshah3008) [inline src/pages/Homepage.js:14]: Just minor suggestion: you can format document later on to make it cleaner

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions and proper folder structure ✅
Added some single comments as suggestion which can be refactored later on 
Rest is all good

File: src/css/cart.css (hunk 4 above)
Comment (saifudinbadani) [reply src/css/cart.css:8]: Yes, will take this into account.

File: src/pages/Homepage.js (hunk 6 above)
Comment (saifudinbadani) [reply src/pages/Homepage.js:14]: I tried using Beautify and Prettier, it messes with code.. will look into other options. Feel free to suggest if you have any.
